golang/Makefile

```diff
@@ -20,13 +20,7 @@ ifeq ($(UNAME), Linux)
 endif
 
 worm.pb.go: ../proto/worm.proto
-	protoc --go_out=plugins=grpc:. --proto_path=../proto ../proto/worm.proto
+	protoc --go_out=plugins=grpc:. --proto_path=../proto ../proto/Command.proto ../proto/Event.proto ../proto/worm.proto
```

Comment:
calling this with just `../proto/worm.proto` doesnt automatically pull the other imports in?

Reply:
It does not properly. I believe it names them as being in a different package.